meta-lmp-base/classes/lmp.bbclass

```diff
@@ -11,6 +11,19 @@ include conf/machine/include/lmp-partner-custom.inc
 # Allow customizations per factory level
 include conf/machine/include/lmp-factory-custom.inc
 
+provision_root_meta () {
+       if [ -f "${SOTA_TUF_ROOT_FETCHER}" ] && [ -x "${SOTA_TUF_ROOT_FETCHER}" ]; then
```

Comment:
These variables are not defined with a standard value in normal lmp environment, so I would add a check if the variable is empty or valid as well.